properties/dapr_nuget.props

```
@@ -1,20 +1,20 @@
 <Project>
   <!-- Nuget package properties when packed using dotnet pack. -->
   <PropertyGroup>
-    <Authors>dapr.io</Authors>
-    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <Owners>Microsoft,Azure-SDK</Owners>
+    <PackageIcon>images\icon.png</PackageIcon>
```

Will this path work? I don't see any image with this path.

I see the path mentioned is 

1. images/webjobs/icon.png
2. images/worker/icon.png

```suggestion
    <PackageIcon>images\webjobs\icon.png</PackageIcon>
```

---

It will work, please see the code below that adds it to images

```xml
  <!-- Icon for nuget packages -->
  <ItemGroup>
    <None Include="$(MSBuildThisFileDirectory)..\images\webjobs\icon.png" Pack="true" Visible="false" PackagePath="images" />
  </ItemGroup>
```